Dockerfile

```diff
@@ -9,15 +9,19 @@ RUN npm ci --no-optional --no-audit --progress=false
 COPY ./wagtailio/static/ ./wagtailio/static/
 RUN npm run build:prod
 
-# Build Python app.
+
+# Build Python app - this stage is a common base for the prod and dev stages
 FROM python:3.8-bullseye AS backend
 
-ENV PYTHONUNBUFFERED=1 \
-    PYTHONPATH=/app \
-    DJANGO_SETTINGS_MODULE=wagtailio.settings.production \
+ARG POETRY_VERSION=1.2.1
+
+ENV DJANGO_SETTINGS_MODULE=wagtailio.settings.production \
+    GUNICORN_CMD_ARGS="--max-requests 1200 --access-logfile -" \
```

Comment:
would be nice to add the jitter param as well (https://adamj.eu/tech/2019/09/19/working-around-memory-leaks-in-your-django-app/#gunicorn) while at it

Reply:
Good idea, done.